<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 176, 177);
background-color: rgb(235, 223, 225);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Insertar</string>
     </attribute>
     <widget class="QLineEdit" name="insertarID">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 245, 220);
background-color: rgb(227, 227, 219);
background-color: rgb(221, 176, 177);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 230, 229);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 230, 229);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>VOLTAJE:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 230, 229);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>POSICIÓN X:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 230, 229);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>POSICIÓN Y:</string>
      </property>
     </widget>
     <widget class="QSlider" name="insertRED">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>41</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 219);
background-color: rgb(221, 176, 177);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="insertBLUE">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>41</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 219);
background-color: rgb(221, 176, 177);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="insertGREEN">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>41</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 219);
background-color: rgb(221, 176, 177);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>220</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 230, 229);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string> RED</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>220</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 230, 229);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string> BLUE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>220</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 230, 229);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>GREEN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarINICIO">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>INICIO</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="insertarFINAL">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>350</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>FINAL</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="insertarVOLT">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 219);
background-color: rgb(221, 176, 177);</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="insertPX">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 219);
background-color: rgb(221, 176, 177);</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="insertPY">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 219);
background-color: rgb(221, 176, 177);</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>insertRED</zorder>
     <zorder>insertBLUE</zorder>
     <zorder>insertGREEN</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>label_7</zorder>
     <zorder>agregarINICIO</zorder>
     <zorder>insertarFINAL</zorder>
     <zorder>insertPX</zorder>
     <zorder>insertPY</zorder>
     <zorder>label</zorder>
     <zorder>insertarID</zorder>
     <zorder>insertarVOLT</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mostrar</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>371</width>
        <height>341</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(242, 245, 237);
background-color: rgb(232, 240, 213);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="mostrar_neuronas">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(122, 156, 87);</string>
      </property>
      <property name="text">
       <string>Mostrar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="guardar_archivo">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(122, 156, 87);</string>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cargar_archivo">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>170</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(122, 156, 87);</string>
      </property>
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-50</y>
        <width>601</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(209, 222, 169);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_9</zorder>
     <zorder>plainTextEdit</zorder>
     <zorder>mostrar_neuronas</zorder>
     <zorder>guardar_archivo</zorder>
     <zorder>cargar_archivo</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Buscar</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>351</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 232, 228);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(160, 178, 194);
color: rgb(8, 31, 92);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Buscar por ID</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Corbel</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 232, 228);
background-color: rgb(233, 232, 228);
color: rgb(8, 31, 92);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>320</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(8, 31, 92);</string>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 214, 235);
background-color: rgb(160, 178, 194);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>320</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 232, 228);
color: rgb(8, 31, 92);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <zorder>label_10</zorder>
     <zorder>tableWidget</zorder>
     <zorder>label_8</zorder>
     <zorder>search</zorder>
     <zorder>label_11</zorder>
     <zorder>spinBox</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Dibujar</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>421</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 225, 210);
background-color: rgb(243, 236, 226);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="dibujar">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(93, 37, 16);</string>
      </property>
      <property name="text">
       <string>Dibujar neuronas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(203, 184, 157);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="lineaas">
      <property name="geometry">
       <rect>
        <x>344</x>
        <y>340</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar lineas</string>
      </property>
     </widget>
     <zorder>label_12</zorder>
     <zorder>dibujar</zorder>
     <zorder>graphicsView</zorder>
     <zorder>lineaas</zorder>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ordenar</string>
     </attribute>
     <widget class="QPlainTextEdit" name="mostrarOrdenar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>351</width>
        <height>341</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 233, 251);
background-color: rgb(255, 244, 254);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ordenar_id">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(156, 10, 149);</string>
      </property>
      <property name="text">
       <string>Ordenar por ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ordenar_voltaje">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>150</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(156, 10, 149);</string>
      </property>
      <property name="text">
       <string>Ordenar por voltaje</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Lista</string>
     </attribute>
     <widget class="QPlainTextEdit" name="textEditListaAdyacencia">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>511</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
